src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -75,6 +76,10 @@ public function evaluate(array $functions, array $values)
 
                 $property = $this->nodes['attribute']->attributes['value'];
 
+                if ($this->nodes['attribute']->isNullSafe) {
+                    return $obj->$property ?? null;
```

Comment:
wait, this is not what nullsafe means. nullsafe is about the left side being null (the object), not the right side (the property)
same issue on the method call below

I guess this needs to be covered by a test case

Reply:
We can simply do this:
```suggestion
                    return $obj?->$property;
```
But then we should exclude throwing a `RuntimeException` from line 73 above it when we use the null-safe syntax!
like this:
```
                if (!\is_object($obj) && !$this->nodes['attribute']->isNullSafe) {
                    throw new \RuntimeException(sprintf('Unable to get property "%s" of non-object "%s".', $this->nodes['attribute']->dump(), $this->nodes['node']->dump()));
                }
```

Reply:
$obj cannot be null here so that won't help